MaterialGraphProject/Assets/UnityShaderEditor/Editor/Data/Implementation/NodeUtils.cs

```diff
@@ -29,7 +30,7 @@ public static void SlotConfigurationExceptionIfBadConfiguration(INode node, IEnu
 
             var toPrint = missingSlots.Select(x => x.ToString());
 
-            throw new SlotConfigurationException(string.Format("Missing slots {0} on node {1}", string.Join(", ", toPrint.ToArray()), node));
+            throw new SlotConfigurationException(String.Format("Missing slots {0} on node {1}", String.Join(", ", toPrint.ToArray()), node));
```

Comment:
Undo this line :) Formatting guide specifies that the aliases like `string` should be used

Reply:
Ok I have done that.